samples/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This sample application provides an example usage of the display ILI9163C from ILITEK.
+The example includes PWM control for backlight dimming and displays text and a counter on the screen.
+
+- Displays "6TRON BY CATIE!" on the screen. 
+- Shows an incrementing counter at the bottom of the screen. 
```

Comment:
Might be better to say it count pasts seconds.

Reply:
Resolved in 1ea59ed680dc881d3afea2fa9cafe1a5cf060ca0.